elements/pl-matching/pl-matching.py

```diff
@@ -77,6 +60,10 @@ def partition(data, pred):
     return (yes, no)
 
 
+def get_chosen_option(option_key, display_options):
+    return display_options[option_key]
+
+
```

Comment:
I'm not sure I see the merit of having a function for this operation.

Reply:
A victim of refactoring... :)